src/models/AnonymousIP.ts

```diff
@@ -0,0 +1,25 @@
+import mmdb = require('maxmind');
+
+/** Class representing the model of a "AnonymousIP" response **/
```

Comment:
Can it say '... **an** "AnonymousIP" response'?

Reply:
Fixed https://github.com/maxmind/GeoIP2-node/pull/7/commits/1ba401d7f46b0a4edc64d945463e54d1dd41376a